assemble/bin/accumulo-service

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function print_usage {
+  cat <<EOF
+Usage: accumulo-service <service> <command>
+
+Services:
+  gc          Accumulo garbage collector
+  monitor     Accumulo monitor
+  master      Accumulo master
+  tserver     Accumulo tserver
+  tracer      Accumulo tracter
+
+Commands:
+  start       Start service
+  stop        Stop service
+  kill        Kill service
+
+EOF
+  exit 1
+}
+
+function invalid_args {
+  echo -e "Invalid arguments: $1\n"
+  print_usage
+  exit 1
+}
+
+function get_host {
+  host="$(hostname -s)"
+  if [[ -z "$host" ]]; then
+    netcmd=/sbin/ifconfig
```

Comment:
`ifconfig` is deprecated, and not installed by default in CentOS 7 and later. The new `ip` command should be used if possible.